### DISCUSSION TITLE
Smplayer: Debian repo signatures are invalid

### DISCUSSION THREAD

**Discussion by fomahub** [smplayer-dev/smplayer#526] [Q&A]
Recently I cannot download updates from the Debian repo.
Warning message:
```
W: GPG error: http://download.opensuse.org/repositories/home:/smplayerdev/Debian_11  InRelease: The following signatures were invalid: EXPKEYSIG 6FD77D5EDF966437 home:smplayerdev OBS Project <home:smplayerdev@build.opensuse.org>
E: The repository 'http://download.opensuse.org/repositories/home:/smplayerdev/Debian_11  InRelease' is not signed.
```
Your help is greatly appreciated.
Tom

**Comment by fomahub** [comment]
Anyone?!

**Comment by dougK44883** [comment]
At least let us know if it's just dead... 
or if it being "looked into"

**Comment by smplayer-dev** [comment]
I really don't know. You can still download a deb package from here:
https://software.opensuse.org/download.html?project=home%3Asmplayerdev&package=smplayer

**Comment by dougK44883** [comment]
Not sure how long you've had support on github - it's been a long time since I've needed support via https://forum.smplayer.info/
I see you have .deb packages here - https://github.com/smplayer-dev/smplayer/releases  it's not as ideal as integrating with the OS, but gitub can notify users of changes.

Any chance you can update buster to bullseye?  current stable.  There are several Ubuntu versions.

**Comment by fomahub** [comment]
Thanks, @dougK44883 and @ricardo, for listening and answers!

> ... .deb packages ... not as ideal as integrating with the OS ...

Indeed, I supposed the developers would examine the signature and publish the appropriate one, sorry.

Greetings,
Tom
